app/models/search.rb

```diff
@@ -212,6 +212,9 @@ def perform_comment_search!
     when :relevance
       # relevance is undefined without search terms so sort by score
       if terms.any?
+        terms_sql = <<~SQL.tr("\n", " ")
```

Comment:
Please rename this variable to `order_sql`. There's a lot going on in this method, I don't want to reuse a variable name and have to keep track of which it is at which line of code.